specificatie/partner.yaml

```diff
@@ -40,37 +26,22 @@ components:
         ontbindingHuwelijkPartnerschap:
           $ref: "#/components/schemas/GbaOntbindingHuwelijkPartnerschap"
     Partner:
-      allOf:
-        - $ref: '#/components/schemas/AbstractPartner'
-        - type: object
-          description: |
-            * **geslachtsaanduiding** - Wordt gevuld op basis van de waarden die voorkomen in de tabel 'Geslacht' uit de Haal-Centraal-BRP-tabellen-bevragen API.
-            * **soortVerbintenis** - Wordt gevuld op basis van de waarden die voorkomen in de tabel  "Soort_Verbintenis" uit de Haal-Centraal-BRP-tabellen-bevragen API.
-          properties:
-            burgerservicenummer:
-              $ref: 'persoon.yaml#/components/schemas/Burgerservicenummer'
-            geslacht:
-              $ref: 'common.yaml#/components/schemas/Waardetabel'
-            soortVerbintenis:
-              $ref: 'common.yaml#/components/schemas/Waardetabel'
-            naam:
-              $ref: 'naam.yaml#/components/schemas/NaamGerelateerde'
-            geboorte:
-              $ref: 'geboorte.yaml#/components/schemas/Geboorte'
-            inOnderzoek:
-              $ref: '#/components/schemas/PartnerInOnderzoek'
-            aangaanHuwelijkPartnerschap:
-              $ref: '#/components/schemas/AangaanHuwelijkPartnerschap'
-            ontbindingHuwelijkPartnerschap:
```

Comment:
waarom heb je ontbindingHuwelijkPartnerschap verwijderd uit de partner?

Reply:
Dit is ook een merge probleem geweest. Ik heb deze er niet bewust uit gehaald. In mijn versie van partner.yaml zit ook geen OntbindingHuwelijkPartnerschapBasis en OntbindingHuwelijkPartnerschapBasisInOnderzoek. Dat hele stuk is er in een ander PR bijgezet. Maar is dus met het oplossen van een merge conflict misgegaan. 

Ik begin nu een beetje huiverig te worden deze PR te mergen.